app/views/mentors/find.njk

```diff
@@ -0,0 +1,63 @@
+{% extends "layouts/main.njk" %}
+
+{% set primaryNavId = "mentors" %}
+
+{% set title = "Enter a teacher reference number (TRN)" %}
+{% set caption = "Add mentor" %}
+
+{% block beforeContent %}
+{{ govukBackLink({
+  text: "Back",
+  href: actions.back
+}) }}
+{% endblock %}
+
+{% block content %}
+
+  {% include "_includes/notification-banner.njk" %}
+
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+
+      {% set headingHtml %}
+        {% include "_includes/page-heading-legend.njk" %}
+      {% endset %}
+
+      <form action="{{ actions.save }}" method="post" accept-charset="utf-8" novalidate>
+
+        {{ govukInput({
+          id: "mentor",
+          name: "mentor[trn]",
+          label: {
+            html: headingHtml,
+            isPageHeading: true,
+            classes: "govuk-label--l"
+          },
+          value: data.mentor.trn,
+          classes: "govuk-input--width-20",
+          errorMessage: errors | getErrorMessage("mentor")
+        }) }}
+
+        {{ govukDetails({
+          summaryText: "I don’t know the teacher reference number",
+          html: 'You can use the <a href="https://find-a-lost-trn.education.gov.uk/">Find a lost teacher reference number (TRN)</a> service to find the TRN for the mentor.'
```

Comment:
html: 'You can use the <a href="https://find-a-lost-trn.education.gov.uk/"> find a lost teacher reference number (TRN)</a> service to find a TRN, or to check the mentor has one. '